### ISSUE TITLE
Next New Comment button should skip over invisible collapsed comments instead of jumping to the top

### ISSUE THREAD

**Issue by talklittle** [theCrius/tildes-extended#38]
## Expected Behavior
Next New Comment button should skip over invisible collapsed comments to the next visible one.

## Current Behavior
If the next new comment is collapsed and invisible, then the button takes you to the top of the page, which is unexpected. Clicking a second time takes you to the next comment after that, as expected.

## Steps to Reproduce
1. Open a topic
2. Wait a while, open it again when topic has new replies
3. Collapse a comment tree; the New comment must be a descendant so when you collapse it, it is completely invisible
4. Click Next New Comment button until you've reached the completely invisible collapsed comment

## Environment
OS: Ubuntu 16.04  
Firefox 61.0.1

**Comment by theCrius** [comment]
Considering that Tildes doesn't store the collapsed status per topic, right now this is not a feature that feel like something to implement.

Also, if you're already scrolling through the topic's comments and collapsing comment's threads, you're already going down "manually" through the comments thus defeating the purpose of the "scroll to new comment" button.

**Comment by talklittle** [comment]
I disagree with the assumption that you have to be going through the topic manually. This extension puts the next new comment at the top of the page. So for example there could be an entire subtree of new comments that I don't care about, immediately following that first one at the top of the page, no additional scrolling necessary to see those. So I hit collapse on the topmost one, and the bug appears.

**Comment by theCrius** [comment]
I see, that could be a valid use case. Thanks for the clarification, i'll review this and try to push it in the next update.
